### PR TITLE
[mobile] Fix the last date of the previous month in the calendar

### PR DESCRIPTION
**Pull request by DohyungLim**
- Fix the value of lastDate.
- Change css opacity to unit.

### CODE REVIEW

File: src/css/profile/mobile/common/calendar.less [hunk 1]
```diff
@@ -84,10 +84,10 @@
 	}
 	
 	&-prev-month-day {
-		opacity: 0.1;
+		opacity: unit(10%) / 100;
```
Comment (TomaszLukawskiSam) [inline src/css/profile/mobile/common/calendar.less]: Let's leave it as it was. The `unit` has applies in specific cases.

Comment (TomaszLukawskiSam) [reply src/css/profile/mobile/common/calendar.less]: eg. to covert `less` variables

File: src/js/profile/mobile/widget/Calendar.js [hunk 2]
```diff
@@ -160,7 +160,7 @@
 				var self = this,
 					ui = self._ui,
 					firstDate = new Date(self._todayYear, self._todayMonth - 1, 0),
-					lastDate = new Date(self._todayYear, self._todayMonth, 0),
+					lastDate = new Date(self._todayYear, self._todayMonth - 1, 0),
```
Comment (TomaszLukawskiSam) [inline src/js/profile/mobile/widget/Calendar.js]: This is little strange. Now, firstDate and lastDate are the same.

File: src/css/profile/mobile/common/calendar.less (hunk 1 above)
Comment (DohyungLim) [reply src/css/profile/mobile/common/calendar.less]: @TomaszLukawskiSam 
I use it in caledar.less, is it better to return the code to the previous one?

File: src/js/profile/mobile/widget/Calendar.js (hunk 2 above)
Comment (DohyungLim) [reply src/js/profile/mobile/widget/Calendar.js]: ok. I remove it. and change exist date value.

File: src/css/profile/mobile/common/calendar.less (hunk 1 above)
Comment (TomaszLukawskiSam) [reply src/css/profile/mobile/common/calendar.less]: I prefer previous, sorry.

File: src/js/profile/mobile/widget/Calendar.js [hunk 3]
```diff
@@ -160,9 +160,8 @@
 				var self = this,
 					ui = self._ui,
 					firstDate = new Date(self._todayYear, self._todayMonth - 1, 0),
-					lastDate = new Date(self._todayYear, self._todayMonth, 0),
 					day = firstDate.getDay(),
-					prevLastDate = lastDate.getDate(),
+					prevLastDate = firstDate.getDate(),
```
Comment (TomaszLukawskiSam) [inline src/js/profile/mobile/widget/Calendar.js]: Probably this is not good way to reach the purpose.

**Review by TomaszLukawskiSam — APPROVED** [review]
Looks good to me.

**Pull request merged**